lib/src/scene.dart

```diff
@@ -107,35 +109,34 @@ abstract class Scene {
   ///   @override
   ///   String get title => 'Counter';
   ///
-  ///   @override
-  ///   List<EnvironmentControlBuilder> get environmentControlBuilders => [
-  ///         (_, VoidCallback rebuildScene) {
-  ///           return StepperControl(
-  ///             title: const Text('Count'),
-  ///             value: count,
-  ///             onDecrementPressed: () {
-  ///               count -= 1;
-  ///               rebuildScene();
-  ///             },
-  ///             onIncrementPressed: () {
-  ///               count += 1;
-  ///               rebuildScene();
-  ///             },
-  ///           );
-  ///         }
-  ///       ];
+  ///  @override
+  ///  final List<EnvironmentControl<Object?>> environmentControls =
+  ///    <EnvironmentControl<Object?>>[
+  ///      StepperControl<int>(
+  ///        title: 'Count',
+  ///        stateKey: 'CounterScene.CountKey',
+  ///        defaultValue: Post.fakePosts().length,
+  ///        onDecrementPressed: (int currentValue) => currentValue - 1,
+  ///        onIncrementPressed: (int currentValue) => currentValue + 1
+  ///      ),
+  ///    ];
```

Comment:
I think it makes sense to give a slightly more realistic example where you assign the stepper control to a typed ivar and then return that from the list here. Because if the developer wants to use the environment control current value in build(), they'll want the typed reference, rather than doing something like `(environmentControls.first() as EnvironmentControl<int>).currentValue`

Reply:
This makes me wonder if `setUp` still needs EnvironmentState as a parameter. WDYT?

Reply:
Removed, let me know if you can think of a compelling reason to add it back.

Reply:
Nah, I agree with removing it! Thanks.